app/src/main/res/layout/activity_main.xml

```diff
@@ -12,13 +12,33 @@
         android:layout_width="300dp"
         android:layout_height="300dp"
         android:layout_gravity="center"
-        android:layout_margin="25dp" />
+        android:layout_margin="10dp" />
```

Comment:
Let's start moving these into a dimens file and let's make all dimens divisible by 4

Reply:
done